### ISSUE TITLE
SQL_ACSII encoding : corrupted accents

### ISSUE THREAD

**Issue by JCornat** [brianc/node-postgres#1033]
Hello,

I have an issue with corrupted accents in a Postgres DB formatted in SQL_ACSII.

What I get in my browser :

> Infirmi�re

What I should get :

> Infirmière

I have tried to change encoding with this workaround, but I always get corrupted accents

```
client.query("SET client_encoding to 'UTF8';" , function(err, result) { //Changing encoding
    client.query(query, params, function(err, result) { //Do the query
        if (err) {
            throw err;
        }
        resolve(result);
    });
});
```

When I use a software like `pgAdmin III`, accents are correctly displayed, so I was wondering what I should do, or if you heard about this problem.

Thanks

**Comment by ederavilaprado** [comment]
I'm with this same scenario and found this same problem...

`Abertura panor�mica.`

**Comment by JCornat** [comment]
I have a workaround for this issue :

```
const data = await requestPostgres(query, params); // Get data from Postgres
data.text = encode(data.text); // Let's imagine text property is corrupted, then encode it

export function encode(data: any): string {
  return (data) ? data.toString('latin1').trim() : data;
}
```

Hope it helps

**Comment by ederavilaprado** [comment]
Hey @JCornat, thanks.

Actually i fixed this and opened a PR: https://github.com/brianc/node-postgres/pull/1338

I tested here and works perfecly for me.
Could you take a look at the PR too and tell me what you think?

Thank you again.

**Comment by JCornat** [comment]
Seems good to me !

Thank you for your PR ! 👍

**Comment by ederavilaprado** [comment]
The code was merged to master, version 6.4.0, and it's working perfectly in my code.

Thanks for the help.
